src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -0,0 +1,6 @@
+{
+  "iris.shaders.reloaded": "Shader aktualisiert!",
+  "iris.keybind.reload": "Aktualisieren der Shader",
+  "iris.keybinds": "Iris",
```

Comment:
Since you're not changing this, don't add it to the translation files because it will automatically fall back to en_us. It's only necessary to add entries for things that differ from en_us.
```suggestion
```